stock_orderpoint_generator/tests/test_orderpoint_generator.py

```diff
@@ -15,7 +15,7 @@ def setUpClass(cls):
         cls.orderpoint_model = cls.env["stock.warehouse.orderpoint"]
         cls.orderpoint_template_model = cls.env["stock.warehouse.orderpoint.template"]
         cls.product_model = cls.env["product.product"]
-        cls.attr = cls.env["product.attribute"].create({"name": "Size"})
+        cls.attr = cls.env["product.attribute"].create({"name": "Test attribute"})
```

Comment:
For avoiding more collapsings:

```suggestion
        cls.attr = cls.env["product.attribute"].create({"name": "stock_orderpoint_generator attribute"})
```

Reply:
thanks!